CONTRIBUTING.md

```diff
@@ -20,6 +20,41 @@ Use the [Discussions](https://github.com/github/gh-gei/discussions) tab in this
 
 All work done by the maintainers of this repo is tracked in this repo using Issues. We have a hierarchical backlog with Epics at the top, broken down into Batches then broken down to Tasks (epic/batch/task is indicated via labels on the issues). You can see an example Epic and navigate down from there [here](https://github.com/github/gh-gei/issues/101).
 
+
+## Debugging code 
+
+There are two ways to debug code within VS code. 
+
+### Run GH-GHEI command locally
+Run the following commands in your terminal depending on the provider you are looking to run the commands for. 
+
+Azure DevOps: 
+`dotnet run --project src/ado2gh/ado2gh.csproj -- [command]`
+
+Bitbucket Server: 
+`dotnet run --project src/bbs2gh/bbs2gh.csproj -- [command]`
+
+GitHub:
+`dotnet run --project src/gei/gei.csproj -- [command]`
+
```

Comment:
A few days ago Arin helped get set up in a C# repl to test file uploads with our [AwsApi](https://github.com/github/gh-gei/blob/main/src/Octoshift/Services/AwsApi.cs). Not sure if this is the right place for it, but it might be useful to others debugging so thought I'd suggest.
````suggestion

### Run code in a C# REPL
In your terminal:
`dotnet tool install --global CSharpRepl --version 0.4.0`
(v0.4.0 is the latest version compatible with .NET 6.0)

Run it:
`csharprepl -r src/bbs2gh/bin/Debug/net6.0/Octoshift.dll`

Then load up assemblies:
```csharp
#r "Octoshift.dll"

// You might need others, for example the AWS SDK:
#r "AWSSDK.Core.dll"
#r "AWSSDK.S3.dll"

using OctoshiftCLI.Services;
```

````

Reply:
This is going to be a great addition, just a few notes:
1 . Add some introduction, e.g. 

You can use a C# REPL to execute any C# code. There are many C# REPLs available, [CSharpRepl](https://fuqua.io/CSharpRepl/) is one of them and can be installed globally with the following:

`dotnet tool install --global CSharpRepl`

2. Not sure if the version is required.
3. I would add a comment before `using OctoshfitCLI.Services;`, something like this:
```csharp
// Add necessary usings
using OctoshiftCLI.Services

// Instantiate your classes
var aws = new AwsApi("access-key-id", "secret-access-key");
```

Reply:
Love this addition! Could you give a little more context of the situation it wold be used in?

Reply:
@begonaguereca, it's a general purpose tool just like `irb` that gives you an interactive console to do whatever you like, for example instantiating `AwsApi` to test file i/o, etc.

Reply:
Thank you! Ok I just merged both of your suggestions together <3